i18n_variable/i18n_variable.module

```diff
@@ -13,7 +13,11 @@ define('I18N_VARIABLE_LANGUAGE_TYPE', 'i18n_language_variable');
  * Implements hook_language_init()
  */
 function i18n_variable_language_init() {
-  if (backdrop_multilingual()) {
```

Comment:
The correct function name is in use now - see the latest commits in 1.x-1.x.

This PR should be obsolete now.